build.js

```diff
@@ -80,7 +80,7 @@ const fencedCodeBlockDefinition = (name, identifiers, sourceScope, language, add
 
 	return `fenced_code_block_${name}:
   begin:
-    (^|\\G)(\\s*)(\`{3,}|~{3,})\\s*(?i:(${identifiers.join('|')})(\\s+[^\`~]*)?$)
+    (^|\\G)(\\s*)(\`{3,}|~{3,})\\s*(?i:(${identifiers.join('|')})(\\s*[^\`~]*)?$)
```

Comment:
I believe this will cause false positive if users write a language name like `pythonnotreally` (which we should not highlight as python)

Are there specific sets of characters that are used to start an attribute? Like `{`?

Reply:
From my experience, it's either `{` or `:` but that's probably not a complete list. Still, could be enough for a start.

Reply:
Let's start with that. So either require (`{`, `:`, or a whitespace) before the attributes

Reply:
Done. Not sure that will appease CI, though.